amundsen_common/models/table.py

```diff
@@ -152,7 +152,7 @@ class Table:
     schema: str
     name: str
     tags: List[Tag] = []
-    badges: Optional[List[Tag]] = []
+    badges: List[Badge] = []
```

Comment:
I think we still need optional ? as not every table has badges.

Reply:
I was thinking that similar to tags (which also don't always exist) this can just be an empty list when they are none